src/Browscap/Data/Factory/UseragentFactory.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * This file is part of the browscap package.
+ *
+ * Copyright (c) 1998-2017, Browser Capabilities Project
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+namespace Browscap\Data\Factory;
+
+use Browscap\Data\Useragent;
+use Browscap\Data\Validator\ChildrenData;
+use Browscap\Data\Validator\UseragentData;
+
+/**
+ * Class UaFactory
```

Comment:
Drop this whole phpdoc - 1) the class is wrong 2) `@category` is meaningless 3) `@author` is useless with VCS